Zend/tests/nested_method_and_function.phpt

```diff
@@ -29,11 +29,11 @@ $c();
 ?>
 --EXPECT--
 string(7) "Baz\foo"
-string(7) "Baz\foo"
+string(0) ""
```

Comment:
This one doesn't look right -- for a free-standing function, `__METHOD__` should have the same result as `__FUNCTION__`.